README.md

```diff
@@ -138,7 +138,9 @@ pip install -e .
 ## Example Usage
 
 Example usage of `idrstream` can be found at [example.ipynb](example.ipynb).
-This notebook was run as a python script ([example.py](example.py)).
+This notebook was run as a python script ([example.py](example.py)) and hence has no output for the final cell.
```

Comment:
```suggestion
We ran this notebook as a python script ([example.py](example.py)) and hence the notebook has no output for the final cell.
```

Active (instead of passive voice)